bootstrap.py

```diff
@@ -639,6 +639,26 @@ def get_api_port():
 print "This script is designed to register new systems or to migrate an existing system to a Foreman server with Katello"
 
 
+def check_prerequisite():
+    rpm_sat = "katello-capsule"
```

Comment:
Yes, I forgot about that. (I am so used to `rpm -q satellite` downstream) +1 to what @evgeni says. Just check the upstream names. We can always patch if it changes.